chall02/mcarrete.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import sys
+
+def stringToMorse(text):
+	morse =		{'A' : '.-', 'B' : '-...', 'C' : '-.-.', 'D': '-..', 'E' : '.',
+				'F' : '..-.', 'G' : '--.', 'H' : '....', 'I' : '..', 'J' : '.---',
+				'K' : '-.-', 'L' : '.-..', 'M' : '--', 'N' :  '-.', 'O' : '---',
+				'P' : '.--.', 'Q' : '--.-', 'R' : '.-.',  'S' : '...', 'T' : '-',
+				'U' : '..-', 'V' : '...-', 'W' : '.--', 'X' : '-..-', 'Y' : '-.--',
+				'Z' : '--..', ' ' : ' '}
+	retMorse = ""
+	count = 0
+	error = "usage: ./mcarrete.py <a-zA-Z string>"
+	for a in text:
+		if a in morse:
+			retMorse += morse[a]
+			count += 1
+		else:
+			print(error)
+			return error
+	print(retMorse)
```

Comment:
You could have simplified this code in a more pythonic way:
```suggestion
    if "".join(c for c in text if c in morse.keys()) != txt:
        return print(error)
    print("".join(morse[c] for c in text))
```